app/javascript/components/App.js

```diff
@@ -34,62 +42,107 @@ class App extends React.Component {
       });
   };
 
-  addScore = (points) => {
-    this.setState({score: this.state.score + points } )
+  updateGameState = (points, kill) => {
+    this.setState({ score: this.state.score + points });
+    this.setState({ lives: this.state.lives - kill });
+  };
+
+  resetGame = () => {
+    // save final score
+    // reset score, lives, questions to initial state
+    this.setState({
+      questions: initialState.questions,
+      score: initialState.score,
+      lives: initialState.lives
+    });
+  };
+
+  componentDidMount() {
+    this.indexGames();
   }
 
+  indexGames = () => {
+    fetch("http://localhost:3000//games")
```

Comment:
```suggestion
    fetch("http://localhost:3000/games")
```
I know it still works with the extra slash but it should also woke with one 🗡️